src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -114,6 +116,20 @@ winrt::com_ptr<Profile> Profile::CopySettings(winrt::com_ptr<Profile> source)
     profile->_BackgroundImageAlignment = source->_BackgroundImageAlignment;
     profile->_ConnectionType = source->_ConnectionType;
 
+    if (source->_UnfocusedAppearance)
+    {
+        if (source->_UnfocusedAppearance.value() == nullptr)

```

Comment:
This if-statement will never get hit because you check if it's set right above it.

Reply:
This is an std::optional<> containing a nullable type.

Pankaj, Prefer disambiguating with `.has_value()` on 119